ft_prepare_mesh.m

```diff
@@ -22,12 +22,24 @@
 %   cfg.downsample  = integer number (default = 1, i.e. no downsampling), see FT_VOLUMEDOWNSAMPLE
 %   cfg.spmversion  = string, 'spm2', 'spm8', 'spm12' (default = 'spm8')
 %
-% For method 'headshape you should specify
+% For method 'headshape' you should specify
 %   cfg.headshape   = a filename containing headshape, a Nx3 matrix with surface
 %                     points, or a structure with a single or multiple boundaries
 %
 % For method 'cortexhull' you should not give input data, but specify
-%   cfg.headshape   = sting, filename containing the pial surface computed by freesurfer recon-all
+%   cfg.headshape   = string, filename containing the pial surface computed by freesurfer recon-all
+%
+% For method 'fittemplate' you should specify
+%   cfg.headshape   = a filename containing headshape
+%   cfg.template    = a filename containing headshape
+%   Example use:
```

Comment:
it is not customary to have examples in FieldTrip help, we have the website for that. Furthermore, this makes the examples even more a mess (by having them on multiple locations)

Reply:
Ok, I will remove the example.

Reply:
the example is still here...